airlock/business_logic.py

```diff
@@ -1215,6 +1279,16 @@ def group_comment(
 
         bll._dal.group_comment(release_request.id, group, comment, user.username, audit)
 
+        if release_request.status != RequestStatus.PENDING:
```

Comment:
I think we might want to nix this for now.

It looks like output-checker comments may need to be private until both checkers have completed their review, and then confirmed to send to comments to user.  So I think we should probably punt on the notification of comments for now, until that flow is clearer?

Reply:
OK, sure. Easy enough to add it back in later